src/k8s-extension/azext_k8s_extension/partner_extensions/OpenServiceMesh.py

```diff
@@ -107,58 +105,58 @@ def Delete(self, client, resource_group_name, cluster_name, name, cluster_type):
+#     subscription_id = get_subscription_id(cmd.cli_ctx)
+#     resources = cf_resources(cmd.cli_ctx, subscription_id)
 
-    cluster_resource_id = '/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.Kubernetes' \
-        '/connectedClusters/{2}'.format(subscription_id, cluster_resource_group_name, cluster_name)
+#     cluster_resource_id = '/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.Kubernetes' \
+#         '/connectedClusters/{2}'.format(subscription_id, cluster_resource_group_name, cluster_name)
 
-    resource = resources.get_by_id(cluster_resource_id, '2020-01-01-preview')
-    cluster_distro = resource.properties['distribution'].lower()
+#     resource = resources.get_by_id(cluster_resource_id, '2020-01-01-preview')
+#     cluster_distro = resource.properties['distribution'].lower()
 
-    if cluster_distro == "general":
-        logger.warning('Unable to determine if distro has been tested for microsoft.openservicemesh, '
-                       'kubernetes distro: \"general\"')
-        return
+#     if cluster_distro == "general":
+#         logger.warning('Unable to determine if distro has been tested for microsoft.openservicemesh, '
+#                        'kubernetes distro: \"general\"')
+#         return
 
-    tested_distros = _get_tested_distros(extension_version)
+#     tested_distros = _get_tested_distros(extension_version)
 
-    if tested_distros is None:
-        logger.warning(field_unavailable_error)
-    elif cluster_distro not in tested_distros.split():
-        logger.warning('Untested kubernetes distro for microsoft.openservicemesh, Kubernetes distro is %s',
-                       cluster_distro)
+#     if tested_distros is None:
+#         logger.warning(field_unavailable_error)
+#     elif cluster_distro not in tested_distros.split():
+#         logger.warning('Untested kubernetes distro for microsoft.openservicemesh, Kubernetes distro is %s',
+#                        cluster_distro)
 
 
-def _get_tested_distros(chart_version):
+# def _get_tested_distros(chart_version):
 
-    try:
-        chart_arc = ChartBuilder({
-            "name": OpenServiceMesh.CHART_NAME,
-            "version": str(chart_version),
-            "source": {
-                "type": "repo",
-                "location": OpenServiceMesh.CHART_LOCATION
-            }
-        })
-    except VersionError:
-        raise InvalidArgumentValueError(
-            "Invalid version '{}' for microsoft.openservicemesh".format(chart_version)
-        )
+#     try:
+#         chart_arc = ChartBuilder({
+#             "name": OpenServiceMesh.CHART_NAME,
+#             "version": str(chart_version),
+#             "source": {
+#                 "type": "repo",
+#                 "location": OpenServiceMesh.CHART_LOCATION
+#             }
+#         })
+#     except VersionError:
+#         raise InvalidArgumentValueError(
+#             "Invalid version '{}' for microsoft.openservicemesh".format(chart_version)
+#         )

```

Comment:
I'd like to ask why we need to block those code?

Reply:
We are trying to change this eventually; however, for now since this is partner code, I didn't want to completely remove it but instead mark it as unused

Reply:
OK